osu.Game/Screens/Edit/Compose/Components/SelectionBoxButton.cs

```diff
@@ -71,5 +72,14 @@ protected override void OnHoverLost(HoverLostEvent e)
         }
 
         public LocalisableString TooltipText { get; }
+
+        internal void TriggerAction()
```

Comment:
This should be public

Reply:
Is there really any reason for it to be public? The action shouldn't really get triggered from anything but the `SelectionBox` or the button itself, so internal does make sense here imo

Reply:
The `TriggerClick` method is `public`, so it doesn't make sense to give less visibility to the more correct `TriggerAction` method. If any ruleset code or test code in the future wants to trigger one of these buttons then it should use `TriggerAction` instead of `TriggerClick` which should be `public` to allow that.

Reply:
`TriggerClick` is a method on the `Drawable`  class meant to be useable from anywhere, so of course that is public. 
That being said, the ability to call it in tests *does* seem like a good reason to make it public that I didn't consider.